src/jsonUtils/sketchImpl/findFontName.ts

```diff
@@ -94,9 +94,12 @@ export const findFont = (style: TextStyle): NSFont => {
   const defaultFontSize = 14;
 
   const fontSize = style.fontSize ? style.fontSize : defaultFontSize;
-  let fontWeight = style.fontWeight
-    ? FONT_WEIGHTS[style.fontWeight.toLowerCase()]
-    : defaultFontWeight;
+  let fontWeight =
+    style.fontWeight &&
+    String(style.fontWeight).toLowerCase() &&
+    FONT_WEIGHTS[String(style.fontWeight).toLowerCase()]
+      ? FONT_WEIGHTS[String(style.fontWeight).toLowerCase()]
+      : defaultFontWeight;
```

Comment:
would this be better?

```suggestion
  let fontWeight = FONT_WEIGHTS[String(style.fontWeight).toLowerCase()] || defaultFontWeight;
```

Reply:
yeah looks good